src/pages/HabitTracker.tsx

```diff
@@ -1,6 +1,83 @@
 import "bootstrap/dist/css/bootstrap.min.css";
 import { links } from "../App";
 import { DashboardCard } from "../components/DashboardCard";
+import styled from "styled-components";
+
+// styled-component styles for Habit Tracker Page
+
+const PageBackground = styled.div`
+  min-height: 100vh;
+  width: 100vw;
+  background-color: #f5f5f5;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 20px;
+`;
+
+const DashboardContainer = styled.div`
+  display: flex;
+  flex: 1;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+
+  @media (max-width: 1100px) {
+    max-width: 100vw;
+  }
+`;
+
+const DashboardCardContainer = styled.div`
+  flex: 1;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  padding: 20px;
+
+  @media (max-width: 1100px) {
+    margin-left: 0;
+    padding: 10px;
+  }
+
+  @media (max-height: 700px) {
+    padding: 10px;
+  }
+`;
+
+const StyledCard = styled(DashboardCard)`
```

Comment:
all of these styled components are also defined in the **Dashboard.tsx**.

You can define each of these components in the `/components` folder and then use it in Dashboard.tsx and HabitTracker.tsx.

Remember: We want to avoid repetition of code.